pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -1104,9 +1108,13 @@ func isPodResourcesResizedDown(pod *v1.Pod) bool {
 func (p *PriorityQueue) AssignedPodUpdated(logger klog.Logger, oldPod, newPod *v1.Pod) {
```

Comment:
Does this include changes to Pod status? or only Pod spec?

Reply:
It contains any changes to assigned Pods, not only spec.